cypress/integration/app.spec.ts

```diff
@@ -0,0 +1,132 @@
+/// <reference types="cypress" />
+
+describe('App', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('visualize default NXdata group as NxImage', () => {
```

Comment:
This one should be under NeXuS context.

Also, we should explicity call a `click` on `source.h5` because the item initially selected may not be the root if `REACT_APP_DEFAULT_PATH` was set in the `.env`.